### ISSUE TITLE
Option to change tag name

### ISSUE THREAD

**Issue by halloichbingunnar** [daeh/zotero-obsidian-citations#14]
Hi, is there a possibility to change the tag ObsCite attaches to the Zotero items?

Use case: I use the coloured tags in Zotero to indicate the status of the item. It moves from red="to read" or "to skim" to orange="read" and eventually green="finished notes" or violet="worth a second look". 
It would be really nice if it was possible to integrate the ObsCite tag into that workflow smoothly :)

Thank you!

**Comment by daeh** [comment]
Hi @halloichbingunnar  - it's certainly possible to add a setting that lets you specify a string other than `ObsCite` for the tag name. Is that all that you're suggesting? Or are you asking if ZotObsCite can dynamically update the tag name based on the status of the item? If the later, where is the status indicated (e.g. in the metadata of the MD note)?

**Comment by halloichbingunnar** [comment]
Haha, actually, that was all I was asking for 🙃 – but the other option sounds interesting, too! So in the second option, the standard tag ObsCite attaches to a zotero item could be changed by including a section in a yaml header?

```---
title: "Confused note about difficult text"
author: "Gunnar"
obscite: "unfinished note"
---```

Or could it even attach more than one tag? 
Or – now I am going really far from your original purpose – could it even read the tags of the Zotero item and add them to the md? (that I did by hand so far and it is somewhat annoying 🌞)

**Comment by daeh** [comment]
Let's make this issue specific to the first idea, since that's really straight forward to implement. If at some point there's a clear use case for a dynamic tag I can think about it more, but I don't currently see much utility in it. I'll aim to put an option to customize the tag name into the next release.

**Comment by daeh** [comment]
@halloichbingunnar will you give `v0.0.15a` a go and see if the custom tag works for you? https://github.com/daeh/zotero-obsidian-citations/releases/download/v0.0.15a/zotero-obsidian-citations-0.0.15a.xpi

**Comment by halloichbingunnar** [comment]
works fine, thank you!

**Comment by daeh** [comment]
closed by 929a3b1359b05c06e836863e42a20a80f38f4322
